tests/RequirementsTest.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import unittest
+
+from coalib.misc.Shell import run_shell_command
+from packaging import version
+
+
+class coalaaCITest(unittest.TestCase):
+
+    def test_find_no_issues(self):
+        with open(os.path.dirname(__file__) + '/../requirements.txt') as f:
```

Comment:
Use os.path.join to join the directories of the path, sth like

os.path.join(os.path.dirname(__file__), os.pardir, 'requirements.txt')